CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## HEAD
 
+* Added contexts feature. Thanks to @apurvis.
```

Comment:
whatever you prefer; i'm just used to always having the next version being the first line of the changelog